### ISSUE TITLE
Image crop

### ISSUE THREAD

**Issue by G-200010** [h751410234/RemoteSensingTeacher#4]
I have a small question. You mentioned that you crop the Dota and xView to 800*800 with overlap of 100, and I found a crop method in this [link](https://github.com/CAPTAIN-WHU/DOTA_devkit/blob/master/ImgSplit.py) of Dota and a crop method in this [link](https://github.com/DIUx-xView/data_utilities) of  xView.  Could you please give me some help of your experiment details of image crop? Thanks a lot.

**Comment by h751410234** [comment]
（1）https://github.com/CAPTAIN-WHU/DOTA_devkit/blob/master/ImgSplit.py is a classic image slicing tool. You can adjust some hyperparameters according to the requirements of our paper to achieve cropping. Specifically, set gap=100 and subsize=800, and leave other parameters as default. This tool can handle both DOTA and xView datasets (some simple code modifications might be needed).
（2）If you want to quickly run through our code, you can try using UCAS-AOD and CARPK for experiments in other domain adaptation directions. This does not require any slicing processing, only simple dataset formatting is needed.

**Comment by G-200010** [comment]
Really appreciate for your reply. I think I got your method. I will try it later. I will also try the simple one. Thanks a lot.

**Comment by G-200010** [comment]
I use the code you mentioned to crop the image and select 3 categories, but I got 5680 images in Dota train set, do you use some strategies in this process? Hope for your reply.

**Comment by h751410234** [comment]
I don't recall doing any additional processing. Are you sure you're using the Dota1.0 dataset and have deleted all images that do not include the three selected categories?

**Comment by h751410234** [comment]
I will upload my processed dataset to Baidu Netdisk in the next few days to facilitate its use and the reproduction of results by everyone.

**Comment by G-200010** [comment]
I think I have done that, I first crop the images, then turn it to coco format, and select three categories during turning to coco format.

**Comment by G-200010** [comment]
I use Dota 1.0 and task 2.

**Comment by h751410234** [comment]
My processing steps are as follows:（1）Slice the large images.（2）Filter the annotations, retaining only those that include the three selected categories.（3）Delete images without any annotations.（4）Convert the dataset into the COCO format.
I'm not sure if this matches your processing steps.

Regarding the selection of annotations, the version used is the original Dota1.0 annotation version, which includes 15 categories. I am not sure about the task-specific labels.

I will upload the download link for the prepared dataset to GitHub in the next few days to facilitate reproducing our results.

**Comment by G-200010** [comment]
Okay. Thanks for your reply.

**Comment by G-200010** [comment]
Could you please tell me just after slicing the large images, how many images do you get?

**Comment by h751410234** [comment]
According to my records, the training set contains 22,381 images and the test set contains 7,638 images.

**Comment by G-200010** [comment]
Really appreciate for your reply. We got the same number, I will check my code after slicing the large images.

**Comment by G-200010** [comment]
Could you please give me some help (maybe the code) and details in selecting 3 categories. I try the pre-trained model of xView to Dota and get the following results. And I think the reason is that I do not select categories right. I got 1864 images for Dota val set. Hope for your reply.
![Uploading hhhhhhhhhhhhhhjsdnaskcsdc.png…]()

**Comment by G-200010** [comment]
Could you please give me some help (maybe the code) and details in selecting 3 categories. I try the pre-trained model of xView to Dota and get the following results. And I think the reason is that I do not select categories right. I got 1864 images for Dota val set. Hope for your reply.
<img width="484" alt="hhhhhhhhhhhhhhjsdnaskcsdc" src="https://github.com/h751410234/RemoteSensingTeacher/assets/172945648/21f9193c-3c9b-4a13-9028-f2d3dbdcd030">

**Comment by G-200010** [comment]
Could you please give me some help (maybe the code) and details in selecting 3 categories. I try the pre-trained model of xView to Dota and get the following results. And I think the reason is that I do not select categories right. I got 1864 images for Dota val set. Hope for your reply.
<img width="1239" alt="hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh" src="https://github.com/h751410234/RemoteSensingTeacher/assets/172945648/fc9ceca5-1fc7-44ff-8073-0bcbb5aec738">

**Comment by G-200010** [comment]
And I get this error when using 
python evaluation.py --config_file {CONFIG_FILE} --opts EVAL True SSOD.RESUME_EMA {CHECKPOINT_FILE}
<img width="452" alt="gggggggggggggggggggg" src="https://github.com/h751410234/RemoteSensingTeacher/assets/172945648/8134efcf-c11d-4dff-8c9a-ee342fc12ab7">

**Comment by h751410234** [comment]
（1）I have uploaded my prepared dataset to Baidu Netdisk, and you can download it for use.
（2）The xView2DOTA model we released is not the EMA model, so it will produce an error. You can use this command for evaluation: python evaluation.py --config_file {CONFIG_FILE} --opts EVAL True RESUME {CHECKPOINT_FILE}

**Comment by G-200010** [comment]
Thanks a lot.
